CITATION.cff

```diff
@@ -0,0 +1,43 @@
+cff-version: 1.2.0
+title: Segway
+version: 3.0.3
+date-released: "2020-01-22"
+message: If you use this software, please cite both the article from preferred-citation and the software itself.
+abstract: This software implements a method of semi-automated genome annotation
+authors:
+  - family-names: Hoffman
+    given-names: Michael M.
+    affiliation: "Princess Margaret Cancer Centre"
+    orcid: "https://orcid.org/0000-0002-4517-1562"
+license: GPL-2.0-only
+repository-code: "https://github.com/hoffmangroup/segway"
+url: "https://segway.hoffmanlab.org"
+preferred-citation:
+  type: edited-work
+  title: Unsupervised pattern discovery in human chromatin structure through genomic segmentation
+  year: 2012
+  authors:
+    - family-names: Hoffman
+      given-names: Michael M.
+      affiliation: "Princess Margaret Cancer Centre"
```

Comment:
Affiliation should be at time of publication (e.g. "University of Washington" for Orion and me)